src/jit/compiler.h

```diff
@@ -1164,7 +1164,13 @@ struct fgArgTabEntry
     regNumber otherRegNum; // The (second) register to use when passing this argument.
 
     SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
-#endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+#elif defined(_TARGET_X86_)
+    __declspec(property(get = getIsStruct)) bool isStruct;
```

Comment:
It's not a brand new member - it was already defined for `FEATURE_UNIX_AMD64_STRUCT_PASSING`, but I didn't want to add the actual field for the x86 case, since it's not needed (I'm not totally sure it's really needed in any case). I'd just as soon not make changes to all the code that uses this - though I wouldn't object to doing so in a separate PR.

---
In reply to: [79510062](https://github.com/dotnet/coreclr/pull/7252#discussion_r79510062) [](ancestors = 79510062)